{"updated_at": "2011-06-29T16:42:41Z", "repo": "rails", "created_at": "2011-06-29T16:42:41Z", "owner": "rails", "url": "https://api.github.com/repos/rails/rails/issues/comments/1468459", "id": 1468459, "issue_url": "https://api.github.com/repos/rails/rails/issues/1903", "issue_id": 1903, "user": {"repos_url": "https://api.github.com/users/josevalim/repos", "type": "User", "html_url": "https://github.com/josevalim", "starred_url": "https://api.github.com/users/josevalim/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/josevalim/subscriptions", "following_url": "https://api.github.com/users/josevalim/following{/other_user}", "gravatar_id": "e837f6b7fd146ab16ed3d663476c063e", "organizations_url": "https://api.github.com/users/josevalim/orgs", "url": "https://api.github.com/users/josevalim", "gists_url": "https://api.github.com/users/josevalim/gists{/gist_id}", "avatar_url": "https://2.gravatar.com/avatar/e837f6b7fd146ab16ed3d663476c063e?d=https%3A%2F%2Fidenticons.github.com%2Fe20b21ae6508f22fc189c60a0880d0b8.png", "id": 9582, "events_url": "https://api.github.com/users/josevalim/events{/privacy}", "login": "josevalim", "received_events_url": "https://api.github.com/users/josevalim/received_events", "followers_url": "https://api.github.com/users/josevalim/followers"}, "body": "That's why the test is wrong. Because if instead of respond_to?() you *call* the method, you will see it will work. The method wants to check if the method is defined, not respond to.", "_id": {"$oid": "52384c5cbd3543c15101142a"}, "html_url": "https://github.com/rails/rails/pull/1903#issuecomment-1468459"}
